### ISSUE TITLE
Can't access account that had been previously tied to my google account

### ISSUE THREAD

**Issue by Ritchieluk** [stolksdorf/homebrewery#555]
Has the google account access been disabled? I cannot sign in with my gmail and I have lost a lot of content that I was in the process of editing. Has this been moved to a new username? Or were those removed? Any help is appreciated.

### Additional Details

**Share Link** :

or

**Brew code to reproduce** : <details><summary>Click to expand</summary><code><pre>

PASTE BREW CODE HERE

</pre></code></details>

**Comment by calculuschild** [comment]
Absolutely nothing has changed on the website. There have been no code
updates. I logged in myself using my google account just now and it worked
fine.

Does your username need to have have "@gmail.com" at the end or not? That's
been a common error for some people depending which way they typed it in
when the account was first made.

**Comment by Ritchieluk** [comment]
I was able to access it from my home laptop, just because my account is automatically logged in. I think it was just an issue from the other laptop I was trying to reach it from. I didn't have the typical drop down that offered google or facebook login when pressing the login button in top right. It took me to a new page entirely that just said "Natural Crit" and then had two entry queries, one for username and one for password. I had never seen that before and my google username/email and password didn't work. 

Either way, my problem is solved for now, I just was very confused by the alternate login that I ran into. I am still unsure what caused that.
